### DISCUSSION TITLE
Class hierarchy, references - Incremental Build

### DISCUSSION THREAD

**Discussion by gquerret** [Riverside-Software/pct#463] [Ideas]
-*-

**Comment by kenherring-rkt** [comment]
Hey @gquerret !  I submitted #466 which "solves" the problem for my tests cases.  I don't know that this full covers the other case you mentioned in issue #455, and I'm not sure the solution is the same you were dreaming of.  Please review and provide any feedback which you think would be useful for me to make additional updates.

**Reply by gquerret** [reply]
I'll have a look at the PR, thanks for your work on PCT !
